src/annotator/config/test/settings-test.js

```diff
@@ -395,4 +395,47 @@ describe('annotator/config/settingsFrom', () => {
       });
     });
   });
+
+  describe('#sideBySide', () => {
+    [
+      {
+        input: 'foo',
+        expectedResult: { mode: 'auto' },
```

Comment:
It is more common to use `expected` as the identifier for the expected return value, in cases where it is clear what this refers to.